sdk/nodejs/output.ts

```diff
@@ -70,9 +70,6 @@ class OutputImpl<T> implements OutputInstance<T> {
      */
     public readonly resources: () => Set<Resource>;
 
-    public readonly apply: <U>(func: (t: T) => Input<U>) => Output<U>;
-    public readonly get: () => T;
```

Comment:
i made these actual instance methods.  I don't see any real value in these being fields.  Plus, it means we don't have to worry about capturing constructor parameters which can be super confusing.